### ISSUE TITLE
Select / ComboBox - isInvalid prop does not add aria-invalid attribute

### ISSUE THREAD

**Issue by endaquigley** [adobe/react-spectrum#8382]
### Provide a general summary of the issue here

Passing `isInvalid` to the `Select` or `ComboBox` component does not add the `aria-invalid` attribute.

Doing the same for the `TextField` component does add the `aria-invalid` attribute.

### 🤔 Expected Behavior?

`aria-invalid` should be set when `isInvalid` is true.

### 😯 Current Behavior

`data-invalid` is set when `isInvalid` is true, but not `aria-invalid`.

### 🖥️ Steps to Reproduce

1. Render a `Select` component with the `isInvalid` prop.
2. Check what attributes are rendered to the DOM.

### Version

1.10.0

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

macOS 15.2

**Comment by reidbarber** [comment]
This is the expected behavior. See https://github.com/adobe/react-spectrum/issues/7605
